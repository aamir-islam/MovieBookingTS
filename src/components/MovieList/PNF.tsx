import React from "react";
import PNFimg from "../Assets/pageNotFound.png";
import { HeadingPNF, PNFImage, PNFMainDiv } from "../Styles/PNF.style";


type Props = {};

const PNF = (props: Props) => {
  return (
    < PNFMainDiv>
    <HeadingPNF>Sorry,there is no result for keyword you searched</HeadingPNF>
      <PNFImage src={PNFimg} alt="pnf" />
    </ PNFMainDiv>
  );
};

export default PNF;
