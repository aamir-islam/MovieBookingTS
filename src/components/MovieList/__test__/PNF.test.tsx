import React from 'react';
import { render, screen } from '@testing-library/react';
import PNF from '../PNF';
import pnf from '../../Assets/pageNotFound.png'

test("Heading testing" , ()=>{
    render(<PNF />)
    const heading = screen.getByText(/Sorry,there is no result for keyword you searched/i)
    expect(heading).toBeInTheDocument()
})
test("Testing for the image" , ()=>{
    render(<PNF />)
    const logo = screen.getByRole<HTMLImageElement>('img') ; 
    expect(logo).toBeInTheDocument()
    expect(logo).toHaveAttribute('alt', 'pnf');
    expect(logo).toHaveAttribute('src', pnf);
})
