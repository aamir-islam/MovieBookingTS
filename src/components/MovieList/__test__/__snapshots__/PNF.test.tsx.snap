// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing page not found snapshot for PageNotFound component 1`] = `
<div>
  <article
    class="sc-bczRLJ iOnNGm"
  >
    <h2
      class="sc-gsnTZi eDgErO"
    >
      Sorry,there is no result for keyword you searched
    </h2>
    <img
      alt="pnf"
      class="sc-dkzDqf cUotN"
      src="pageNotFound.png"
    />
  </article>
</div>
`;
